schema/draft/schema.json

```diff
@@ -0,0 +1,2121 @@
+                        "properties": {},
+                        "type": "object"
+                    },
+                    "description": "Experimental, non-standard capabilities that the client supports.",
+                    "type": "object"
+                },
+                "roots": {
+                    "description": "Present if the client supports listing roots.",
+                    "properties": {
+                        "listChanged": {
+                            "description": "Whether the client supports notifications for changes to the roots list.",
+                            "type": "boolean"
+                        }
+                    },
+                    "type": "object"
+                },
+                "sampling": {
+                    "additionalProperties": true,
+                    "description": "Present if the client supports sampling from an LLM.",
+                    "properties": {},
+                    "type": "object"
+                }
+            },
+            "type": "object"
+        },
+        "ClientNotification": {
+            "anyOf": [
+                {
+                    "$ref": "#/definitions/CancelledNotification"
+                },
+                {
+                    "$ref": "#/definitions/InitializedNotification"
+                },
+                {
+                    "$ref": "#/definitions/ProgressNotification"
+                },
+                {
+                    "$ref": "#/definitions/RootsListChangedNotification"
+                }
+            ]
+        },
+        "ClientRequest": {
+            "anyOf": [
+                {
+                    "$ref": "#/definitions/InitializeRequest"
+                },
+                {
+                    "$ref": "#/definitions/PingRequest"
+                },
+                {
+                    "$ref": "#/definitions/ListResourcesRequest"
+                },
+                {
+                    "$ref": "#/definitions/ReadResourceRequest"
+                },
+                {
+                    "$ref": "#/definitions/SubscribeRequest"
+                },
+                {
+                    "$ref": "#/definitions/UnsubscribeRequest"
+                },
+                {
+                    "$ref": "#/definitions/ListPromptsRequest"
+                },
+                {
+                    "$ref": "#/definitions/GetPromptRequest"
+                },
+                {
+                    "$ref": "#/definitions/ListToolsRequest"
+                },
+                {
+                    "$ref": "#/definitions/CallToolRequest"
+                },
+                {
+                    "$ref": "#/definitions/SetLevelRequest"
+                },
+                {
+                    "$ref": "#/definitions/CompleteRequest"
+                }
+            ]
+        },
+        "ClientResult": {
+            "anyOf": [
+                {
+                    "$ref": "#/definitions/Result"
```

Comment:
According to `schema.ts`, this line should say `.../EmptyResult`, not `.../Result`.